ambry-router/src/main/java/com/github/ambry/router/PutOperation.java

```diff
@@ -1772,6 +1773,7 @@ private class MetadataPutChunk extends PutChunk {
     private final TreeMap<Integer, Pair<StoreKey, Long>> indexToChunkIdsAndChunkSizes;
     private int intermediateChunkSize = routerConfig.routerMaxPutChunkSizeBytes;
     private Pair<? extends StoreKey, BlobProperties> firstChunkIdAndProperties = null;
+    private final PartitionId reservedPartitionId;
```

Comment:
We don't use the reservedPartitionId yet in the PR, right?

Reply:
Yes. We don't use it yet. Right now we want to collect data about
1. How many partitions are RW and PRW and whats the right threshold.
2. How many metadata chunks with reserved ids will fail or need to go cross colo.